### PR TITLE
Building with Responsive Desing  

### PR DESCRIPTION
**Pull request by 0x1100010010**
Built responsive clone of TNW website using

- Grids for header and Dashboard
- Flexbox for Feed sections
- Used Fontawsme icons for icons along with extenal svgs
- Used fixed positioning in Header and Feed profile pictures
- Created Responsive designs for Mobile, Tablet and Desktop using media queries

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,22 +1,22 @@
 ![](https://img.shields.io/badge/Microverse-blueviolet)
 
-# Project Name
+# TNW-Responsive-Clone
 
-> One paragraph statement about the project.
+> Creating a responsive clone of TNW (thenextweb.com) website. Projects includes differnet stylings of screens i.e. Moble UI, Tablet UI, Desktop UI. Dynamic images and content.
 
 ![screenshot](./app_screenshot.png)
```
Comment (MuriloRoque) [inline README.md]: - [x] Fix your screenshot, it's not showing in your README file.

File: index.html [hunk 2]
```diff
@@ -0,0 +1,877 @@
...
+                            <ul>
+                                <li><a href="#" class="credits">Matthew Beedham</a></li>
+                                <li>38 minutes ago</li>
+                            </ul>
+                        </div>
+
+                    </div>
+                    <div class="cards">
+                        <a class="img2" href="#"></a>
+                        <!-- <img src="./assets/images/agile-marketing-gq-540x270.png"> -->
```
Comment (MuriloRoque) [inline index.html]: - [x] Please remove this commented code.

File: index.html [hunk 3]
```diff
@@ -0,0 +1,877 @@
...
+        <div class="main-flex">
+            <section class="latestnews">
+                <div><a href="#" class="orange-headings main-headings">
+                        Latest News >
+                    </a>
+                </div>
+                <div class="cards-flex">
+                    <div class="cards">
+                        <a class="img1" href="#"></a>
+                        <!-- <img src="./assets/images/uber-amsterdam-british-drivers-employee-gdpr-data-car-court-.jpg"> -->
```
Comment (MuriloRoque) [inline index.html]: - [x] Please remove this commented code.

File: styles/mobile.css [hunk 4]
```diff
@@ -0,0 +1,1305 @@
...
+  }
+
+  .posts .cards {
+    height: 15rem;
+  }
+
+  .lat-li li {
+    font-size: 0.8rem;
+
+    /* margin: 20px 20px; */
```
Comment (MuriloRoque) [inline styles/mobile.css]: - [x] Please remove this commented piece of code.

File: styles/mobile.css [hunk 5]
```diff
@@ -0,0 +1,1305 @@
...
+
+  .main-title {
+    visibility: visible;
+    width: 60%;
+    margin: 6% 2%;
+    position: absolute;
+    top: 50px;
+    display: flex;
+
+    /* justify-content: space-between; */
```
Comment (MuriloRoque) [inline styles/mobile.css]: - [x] Please remove this commented piece of code.

File: styles/mobile.css [hunk 6]
```diff
@@ -0,0 +1,1305 @@
...
+  }
+
+  .posts .cards {
+    height: 15rem;
+  }
+
+  .lat-li li {
+    font-size: 0.8rem;
+
+    /* margin: 20px 20px; */
```
Comment (MuriloRoque) [inline styles/mobile.css]: - [x] Please remove this commented piece of code.

**Review by MuriloRoque — CHANGES_REQUESTED** [review]
## Status: Changes requested :red_circle: 

### Hello @bashforger and @helman101 :wave: 

### Great job in this project, it's very good, guys :clap: :clap: 

### Please check my suggestions for improvement below:

## General

- [x] Please check my comments in your code, there are some commented pieces of code you need to delete.

- [x] In your CSS, I detected a bad practice of repeating a lot of the same property. I suggest that you create specific classes for those rules and then apply the classes to the elements you want to have them. For example, create a `d-flex` class with `display: flex`, then give the elements you want 'flexed' in this class. Check below the amount of times you are repeating those rules:

      36 |     display: flex;
      14 |     background-size: cover;
      13 |     flex-direction: column;
      12 |     color: black;
      11 |     linear-gradient(to top, black, rgba(255, 255, 255, 0)),
      11 |     background:
       9 |      justify-content: space-between;
       9 |      text-decoration: none;
       8 |      width: 100%;

## Large screen

- [x] In the 3-article row below the navbar, there should be a TNW logo inside of the big image. Check below:

**Original:**

Screenshot from 2020-10-29 14-37-53


**Yours:**

Screenshot from 2020-10-29 14-37-49

**NOTE:** The logo doesn't appear on medium and small screens!

- [x] In the footer, the Copyright part should be larger, then it gets smaller as you reduce the screen size. Compare it to the original:

**Original:**

Screenshot from 2020-10-29 14-41-36

**Yours:**

Screenshot from 2020-10-29 14-41-39

## Summary

- Very good project, the page is basically the same, with some hard to spot differences :smile: 
- Very professional repository, keep up the good work :muscle: 
- We only need to fix some details and will be ready to move on :rocket: 


### Kindly make the requested changes and submit for another review, please :pray: 

### I will be on Slack 24/7 in case you have any questions about this review :handshake: 

Murilo Roque Paiva da Silva

- Github: [@muriloroque](https://github.com/MuriloRoque)
- Linkedin: [MuriloRoque](https://www.linkedin.com/in/murilo-roque-b1268741/)

[Twitter Follow](https://twitter.com/MuriloRoquePai1)

File: index.html (hunk 2 above)
Comment (0x1100010010) [reply index.html]: Thank you for such a detailed review. I really wasn't expecting such a detailed inspection. This issue is fixed!

File: README.md (hunk 1 above)
Comment (0x1100010010) [reply README.md]: Fixed!

File: styles/mobile.css (hunk 6 above)
Comment (0x1100010010) [reply styles/mobile.css]: Removed!

File: styles/mobile.css (hunk 5 above)
Comment (0x1100010010) [reply styles/mobile.css]: Fixed!

File: styles/mobile.css (hunk 4 above)
Comment (0x1100010010) [reply styles/mobile.css]: Removed!

File: index.html (hunk 3 above)
Comment (0x1100010010) [reply index.html]: Removed!

File: index.html [hunk 7]
```diff
@@ -0,0 +1,875 @@
...
+                    <li><a href="">HARD FORK</a></li>
+                    <li><a href="">PLUGGED</a></li>
+                    <li><a href="">README</a></li>
+                    <li><a href="">GROWTH QUARTERS</a></li>
+                    <li><a href="">SHIFT</a></li>
+                    <li><a href="">NEURAL</a></li>
+                </ul>
+                <div>
+                    <a href=""><img src="./assets/images/search.svg" alt="TNW Logo"></a>
+                    <a href=""><img class="menu-icon" src="./assets/images/menu.svg" style="color: aliceblue;" alt="TNW Logo"></a>
```
Comment (euqueme) [inline index.html]: - [ ] no inline styling
> let's put this: `style="color: aliceblue;"` within the `menu-icon` class in your css

**Review by euqueme — APPROVED** [review]
> ### Pull Request reviewed by Maria Eugenia Queme P. (TSE)

### Hello @bashforger and @helman101 :balloon: :lollipop: 

### Thank you for doing the changes requested by the previous reviewer :medal_sports: :medal_sports:  your project looks amazing and your code is impeccable :sparkles: there are a couple of changes I strongly recommend you do but I think you have done a terrific job already so I won't keep you here any longer I'm approving your project Congratulations! :tada: :tada: :tada: 

## [OPTIONAL]
> I strongly recommend doing the following changes to make your project stand out a lot more from the others :ok_hand: 
### Mobile Screens
#### Latest funding rounds on Index.co
- [ ] This section should only display 3 articles on this screen size (hides one)
- Yours
image
- Original
image
### Large Screens
#### Latest funding rounds on Index.co
- [ ] The menu links in this screen size switch places you could achieve this with `row-reverse` or `column-reverse` `flex-direction` property
- Yours
image
- Original
image
### Bad practices
- [ ] Make sure that basic CSS bad practices are not used ([article for support ](https://speckyboy.com/good-bad-css-practices)) ([screenshot](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/html_css/section/css_bad_practices_requirement.png) from Pathwright)
> I left a comment directly on your HTML code where I spotted inline styling, I strongly recommend you fix it because this was a specific project requirement.

### That's it, team! :raised_hands: 

### Please :pray: do the changes :technologist: :technologist: and merge this branch when you are done :memo: 

### Keep up the good work :rocket: 

Any question? ask me on slack **@ Maru** :woman_technologist: 

PROJECT STATUS: APPROVED :heavy_check_mark:

**Pull request merged**